packages/settings-library/src/settings_library/efs.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+
+from pydantic import Field
+
+from .base import BaseCustomSettings
+
+
+class AwsEfsSettings(BaseCustomSettings):
+    EFS_DNS_NAME: str = Field(
+        description="AWS Elastic File System DNS name",
+        example="fs-xxx.efs.us-east-1.amazonaws.com",
+    )
+    EFS_PROJECT_SPECIFIC_DATA_DIRECTORY: str = Field(default="project-specific-data")
+    EFS_MOUNTED_PATH: Path = Field(
+        default=Path("/data/efs"),
+        description="This is the path where EFS is mounted to the EC2 machine",
+    )
+    EFS_ENABLED_FOR_USERS: list[int] = Field(
+        description="This is temporary solution so we can enable it for specific users for testing purpose",
+        example=[1],
+    )
```

Comment:
Q: Will this go away in the future?
I'm asking this since you are using `in` with a `list`. Which is O(N). With a `set` it would be O(1).

Reply:
Yes, this is for sure temporary, this logic might in the future live in the `group_extra_properties`, should I still change it to set?

Reply:
in my mind set is the appropriate type but since this is a dev-only feature I guess, maybe no need to optimize here. Every junior dev understands lists so there is a virtue in them.

Reply:
changed :+1:

Reply:
It seems there is issue how you define empty set in the .env-devel file? do you have some ideas?
```
E           settings_library.base.DefaultFromEnvFactoryError: 1 validation error for AwsEfsSettings
E           EFS_GUARDIAN_AWS_EFS_SETTINGS -> EFS_ONLY_ENABLED_FOR_USERIDS
E             value is not a valid set (type=type_error.set)
```